lib/cron/synth-switch.ts

```diff
@@ -394,12 +400,12 @@ async function main(metrics: MetricsLogger) {
         };
         return formattedRow;
       });
-      metrics.putMetric(MetricName.SynthOrdersCount, formattedResult.length, Unit.Count);
+      metrics.putMetric(Metric.SYNTH_ORDERS, formattedResult.length, MetricLoggerUnit.Count);
```

Comment:
Is this the total number of synth orders found?

Reply:
yea